linux_os/guide/system/auditing/auditd_configure_rules/file_permissions_var_log_audit/tests/common_0600.sh

```diff
@@ -0,0 +1,10 @@
+# platform = multi_platform_ol,multi_platform_rhel
+#!/bin/bash
```

Comment:
The "shebang" should always be on the first line of a file. This problem repeats below.